site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,100 @@
+1. In the service worker, write the data to a new location.
+
+### Partitioning {: #storage-partitioning }
+
+To prevent certain types of cross-site tracking, [storage partitioning][storage-partitioning]
+creates a seperate storage area in certain third-party contexts. In practice, this means that
+if site A embeds an iframe containing site B, site B will not be able to access the same storage
+it would usually have when navigated to directly.
+
+To mitigate the impact of this in extensions, two key exemptions apply:
+
+- If a page with the chrome-extension:// scheme is embedded in any site, storage partitioning will
+not apply, and the extension will have access to its normal storage partition.
+- If a page with the chrome-extension:// scheme includes an iframe, and the extension has
+[host permissions][declare-permissions] for the site it is embedding, that site will also have
+access to its normal storage partition.
+
+Combined, these mean that storage partitioning is not usually something which extension developers
+should need to consider.
+
+## Cookies {: #cookies }
+
+### Secure cookies {: #secure-cookies }
+
+The [`Secure`][cookies-restrict-access] cookie attribute is only supported for the https:// scheme.
+Consequently, chrome-extension:// pages are not able to set cookies with this attribute. This also
+means that extensions cannot use the [`Partitioned`][chips] attribute, which requires that the
+`Secure` attribute is also set.
+
+### Partitioning {: #cookies-partitioning }
+
+{% Aside %}
+As of 2023, this is an ongoing area of investigation and the behavior here is subject to change. In
+particular, we are currently investigating changing the behaviour when an extension with host
+permissions embeds a third-party site: [https://crbug.com/1463991](https://crbug.com/1463991)
+{% endAside %}
+
+As mentioned [above](#secure-cookies), extensions cannot create partitioned cookies, and cookies
+are not currently partitioned by default (see the Privacy Sandbox
+[timeline][privacy-sandbox-timeline] for related work). Consequently, chrome-extension:// pages
+embedded in other websites can access cookies associated with the extension origin as normal.
+
+If a chrome-extension:// page contains an iframe, any sites it embeds will use the extension
+origin as the partitioning key. This means that they will **not** be able to access partitioned
+cookies associated with other partitions.
+
+Partitioning also has implications for the [chrome.cookies][chrome-cookies] API. For more
+information, see the [API reference][chrome-cookies-partitioning].
+
+[chrome-storage-api]: /extensions/reference/storage
+[offscreen]: /extensions/reference/offscreen
+[on-message]: /docs/extensions/reference/runtime/#event-onMessage
+[create-offscreen]: /docs/extensions/reference/offscreen/#method-createDocument
+[send-message]: /docs/extensions/reference/runtime/#method-sendMessage
+[storage-partitioning]: /docs/privacy-sandbox/storage-partitioning
```

Comment:
This page includes a section on Extensions. I would ensure you also update that location to ensure the messaging is the same. I'd suggest that the entry under the Privacy Sandbox docs should be a more concise summary and links through to this page as the canonical source.